databases/relational-databases.md

```diff
@@ -75,31 +62,27 @@ To understand, query, and insert information into a relational database, a techn
 
 - [Slides](https://drive.google.com/open?id=1xK7_t_yJcu4RcBkj0Gv-t5uyBCNr0g4cHKqAJSxNwY0) | [Video Walkthrough of Slides](https://drive.google.com/file/d/1V0bk3fH_8PsRE3Vz4J3qe3TTiqBClT6y/view)
 
-Also briefly review: [LucidChart: What is a Database Model?](https://www.lucidchart.com/pages/database-diagram/database-models). Be sure to read the [Relational Model](https://www.lucidchart.com/pages/database-diagram/database-models?a=1) section, and briefly skim over the rest just so you're aware that there are many types of database models. The only one we're working with here is the relational model, so don't worry about learning the rest yet.
-
-- Look through the rest of the links in the Materials Section. [SQL Fiddle](http://sqlfiddle.com) is the tool where you can implement your sql knowledge. Practically implement creating a database schema and perform operations on it using the tool.
+- Look through the rest of the links in the Materials Section. [SQL Fiddle](http://sqlfiddle.com) is the tool where you can implement your SQL knowledge. Practically implement creating a database schema and perform operations on it using the tool.
 
-#### Relational Database Basics
+### Relational Database Basics
 
 - The specific problems a database solves
-  - Persistence of information
+  - Storage of information
   - Centralized information
   - Search stored information
-  - Form relationships between sets of information
-- Users of databases
-  - Front end people should have concepts and write very basic SQL
-  - Back end people spend a large amount of time with databases
-- [Structure of a relational database](https://www.lucidchart.com/pages/database-diagram/database-design)
-- Few basic column types
-  - `varchar`
-  - `text`
-  - `integer`
-  - `primary key`, `auto increment`
+  - Form relationships between tables containing information
+  
+- [Few basic column data types](https://www.w3schools.com/sql/sql_datatypes.asp)
+  - `varchar` (Just like **string** available in different programming languages like JS)
+  - `int` (For holding Numeric Values)
+  - `boolean or bool` (For storing true, false or null)
+  - `date` (stores the dates only)
+  - `primary key`
```

Comment:
It's a little confusing to have PK in the same list of data types, as they are different concepts.

I would split it into a separate section explaining what a primary key is, that it must be unique and can be different types but is commonly an auto-increment integer.